src/index.html

```diff
@@ -6,9 +6,87 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+    rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          src="/images/logo.png"
+          alt="Moyo logo"
+          class="logo__img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+            class="nav__link is-active"
```

Comment:
If mentors point out a formatting problem for one tag, then also check the rest of the tags for formatting in accordance with the checklist
![image](https://github.com/mate-academy/layout_moyo-header/assets/54936044/6345417c-a3d8-4b71-b60e-5032046c6c98)